addons/mass_mailing/static/src/js/mass_mailing.js

```diff
@@ -47,4 +48,15 @@ FieldTextHtml.include({
     },
 });
 
+ListRenderer.include({
```

Comment:
I suppose the purpose of this code is to add a trash button instead of a cross to unlink items. Some questions ..
1/ Why is it specific to mass mailing ?
2/ If we really want something specific here couldn't we support a trashIcon parameter to propagate to the list view instead of doing hardcoded JS change ? (cc @ged-odoo )

Bisous.

Reply:
I confess I am a bit glad of your remark, I didn't like that way but didn't dare to change the widget itself for such a customization. I'll rewrite it the clean way as soon as I can!

Reply:
don't be afraid to come visit us in our small house, if you need help.

Reply:
> Note that the code above (on kanban view, and on fieldtexthmtl) is totally wrong as well... This is definitely not the way it should be done

Probably, but just for this small onboarding I propose to not rewrite the mass mailing JS code ;) . I can do another tasks for that specific point.

Reply:
Thanks!